### PR TITLE
Mtd 15 country

### PR DESCRIPTION
**Pull request by ghilyana**

### CODE REVIEW

File: src/main/java/com/ghilly/controller/CountryController.java [hunk 1]
```diff
@@ -22,24 +24,24 @@ public CountryController(CountryService service) {
...
-    public String getCountry(@PathVariable int countryId) {
+    public Country getCountry(@PathVariable int countryId) {
         logger.info("The country with this ID {} is: ", countryId);
         return service.getCountry(countryId);
     }
 
     @PutMapping("/{countryId}")
     public void update(@PathVariable int countryId, @RequestBody String newName) {
-        String oldName = service.getCountry(countryId);
+        String oldName = service.getCountry(countryId).getName();
```
Comment (gregorusanov) [inline src/main/java/com/ghilly/controller/CountryController.java]: not need

File: src/main/java/com/ghilly/controller/CountryController.java [hunk 2]
```diff
@@ -22,24 +24,24 @@ public CountryController(CountryService service) {
...
         service.upgrade(countryId, newName);
```
Comment (gregorusanov) [inline src/main/java/com/ghilly/controller/CountryController.java]: use country class for it

File: src/main/java/com/ghilly/classes/Country.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
+package com.ghilly.classes;
```
Comment (gregorusanov) [inline src/main/java/com/ghilly/classes/Country.java]: rename to model

File: src/main/java/com/ghilly/controller/CountryController.java (hunk 1 above)
Comment (ghilyana) [reply src/main/java/com/ghilly/controller/CountryController.java]: Done

File: src/main/java/com/ghilly/classes/Country.java (hunk 3 above)
Comment (ghilyana) [reply src/main/java/com/ghilly/classes/Country.java]: done

File: src/main/java/com/ghilly/controller/CountryController.java (hunk 2 above)
Comment (ghilyana) [reply src/main/java/com/ghilly/controller/CountryController.java]: Done

**Review by gregorusanov — APPROVED** [review]

**Pull request merged**